pkgs/development/python-modules/uqbar/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, pip
+, black
+, unidecode
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "uqbar";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-cEhWXGtMSXVjT5QigDedjT/lwYQnVqPCE5vbctXWznk=";
+  };
+
+  propagatedBuildInputs = [
+    setuptools
+    pip
+    black
+    unidecode
+    sphinx
+  ];
```

Comment:
We only need spinx, unidecode and black.
https://github.com/josiah-wolf-oberholtzer/uqbar/blob/9029b3c4eeb9460d113c49d2e00bae32c56fd143/pyproject.toml#L21-L25

black is a formatter so it should be added to pythonRemoveDeps. I don't think it is needed in the package.

Reply:
It seems black wasn't required at all to build it at all, despite being declared in the `pyproject.toml` (alongside several other extraneous formatters for test dependencies)